src/DependencyInjection/SchedulerBundleConfiguration.php

```diff
@@ -102,6 +74,130 @@ public function getConfigTreeBuilder(): TreeBuilder
+                            ->values(['shell', 'null', 'http', 'command', 'chained'])
+                        ->end()
+                        ->scalarNode('description')->end()
+                        ->scalarNode('expression')->defaultValue('* * * * *')->end()
+                        ->integerNode('priority')
+                            ->min(-1000)->max(1000)
+                        ->end()
+                        ->booleanNode('single_run')->end()
+                        ->booleanNode('output')->end()
+                        ->booleanNode('output_to_store')->end()
+                        ->integerNode('nice')->end()
+                        ->integerNode('max_executions')->end()
+                        ->integerNode('execution_delay')->end()
+                        ->scalarNode('timezone')->end()
+                        ->scalarNode('tracked')->end()
+                        ->scalarNode('state')->end()
+                        ->floatNode('max_duration')->end()
+                        ->arrayNode('tags')
+                            ->normalizeKeys(false)
+                            ->variablePrototype()->end()
+                        ->end()
+                        ->variableNode('command')->info('The command to run. (shell|command type) ')->end()
+                         ->scalarNode('cwd')->info('The working directory to use. (shell type)')->end()
+                         ->arrayNode('environment_variables')
+                            ->info('environment_variables to passed. (shell type)')
+                            ->useAttributeAsKey('name')
+                            ->normalizeKeys(false)
+                            ->variablePrototype()->end()
+                        ->end()
+                        ->floatNode('timeout')->info('The timeout in seconds to disable. (shell type)')->end()
+                        ->arrayNode('arguments')
+                            ->info('arguments to passed. (command type)')
+                            ->normalizeKeys(false)
+                            ->variablePrototype()->end()
+                        ->end()
+                        ->arrayNode('options')
+                            ->info('options to passed. (command type)')
+                            ->useAttributeAsKey('name')
+                            ->normalizeKeys(false)
+                            ->variablePrototype()->end()
+                        ->end()
+                        ->scalarNode('url')->info('url (http type)')->end()
+                        ->scalarNode('method')->info('HTTP Method. (http type)')->end()
+                        ->arrayNode('client_options')
+                            ->info('HTTP client options. (http type)')
+                            ->useAttributeAsKey('name')
+                            ->normalizeKeys(false)
+                            ->variablePrototype()->end()
+                        ->end()
+                        ->arrayNode('tasks')
```

Comment:
@Guikingone do you know if it is possible to define a recursive configuration? 
the subtasks need (almost) the same configuration as the first level of tasks